### DISCUSSION TITLE
Cannot get Hubs server to install on a Digital Ocean droplet

### DISCUSSION THREAD

**Discussion by vhelin-softability** [mozilla/hubs#3244] [Hubs Cloud Questions] [answered]
Anyone got the Hubs server installed on a DigitalOcean server? I run the installation script, but all I get is 100% CPU usage and nothing happens. Certbot process is spawned, it consumes all CPU and it gets shutdown pretty quickly, and this loops... If I try to run certbot manually it fails, there is no webserver running (certbot needs one?), if I install Apache(2) and run it I cannot access the www-pages... Even if I modify the firewall so UDP and TCP are all passthrough...

I had one droplet in Germany and one in New York, both gave the same results... I've done all the guides tell me to do, and DNS should be ok as I can ping the server using IP and name string, both work, and name translates to the IP...

I also did "apt-get update" & "apt-get updage" for the server as it (the server image) seemed to be a little out-of-date, but no help

![image](https://user-images.githubusercontent.com/63110819/97552186-9e0bb900-19dc-11eb-9860-148a200f22d3.png)

**Comment by yakyouk** [comment]
I think this is the same problem: Certbot generated certs for the domains correctly but the certbot bio service is erroring out: `journalctl -f` :
`certbot.default@polycosm(O): certbot: error: unrecognized arguments: -- --config=/hab/svc/certbot/config/certbot.renew.ini --register-unsafely-without-email --agree-tos`
`certbot.default@polycosm(O): sleep: missing operand `
It can be turned off: `bio svc stop mozillareality/certbot` but I suppose it turns off cert renewal.

**Comment by heaversm** [comment]
I got stuck with the [personal access token step](https://github.com/mozilla/hubs/discussions/3279). Did you have any trouble there?

**Comment by pattersonbl2** [comment, marked as answer]
Did you still having issues with this ? if you found a fix please let me know.

**Reply by vhelin** [reply]
I gave up and moved to AWS where it installed just fine. :)
